### ISSUE TITLE
Unable to access web interface

### ISSUE THREAD

**Issue by uBadRequest** [ArturSS7/TukTuk#13]
When trying to visit port 1234, it appears that new ssl verifications are trying to be performed?
```
echo: http: TLS handshake error from myip:53046: acme/autocert: missing certificate
echo: http: TLS handshake error from myip:50624: 429 urn:ietf:params:acme:error:rateLimited: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/
```
I have a valid certificate, and it works just fine, very confused as to what this is doing

**Comment by ArturSS7** [comment]
Hello. Are you trying to use your own certificate? Because for the admin panel there is no need to generate certificate manually, it will be automatically generated for your domain by the app. Also, you can solve this issue by disabling https in admin panel. To do this you need to comment lines 54 and 87 in backend.go file in backend package and uncomment the line 88.

**Comment by uBadRequest** [comment]
Yeah thats what I ended up doing, but now I can't seem to get any logging to work. My DNS records in cloudflare are identical to yours, I verified that I can insert data into the database with the tuk user, and can view the data in the database from the management portal.

![image](https://user-images.githubusercontent.com/47282747/94314183-05ea6200-ff4e-11ea-84f7-92a8a526c5c0.png)
![image](https://user-images.githubusercontent.com/47282747/94314256-2b776b80-ff4e-11ea-85d9-e71269b74085.png)
![image](https://user-images.githubusercontent.com/47282747/94314304-4ba72a80-ff4e-11ea-8b24-c86863a14820.png)

I tried to see if I could get it to send logs to a discord webhook
![image](https://user-images.githubusercontent.com/47282747/94314449-81e4aa00-ff4e-11ea-8b12-68d5752ae231.png)

This is what I see what I do "go run main.go"
![image](https://user-images.githubusercontent.com/47282747/94314641-ebfd4f00-ff4e-11ea-9e01-b777878ea3e2.png)


Happen to see where I may have made a mistake? I can't figure it out...

I blocked my domain out, don't want to have anyone poking around :smile:

**Comment by ArturSS7** [comment]
In order to make logging work, you need first to generate domain, same as in burp(if you used it). Go to dns management, generate new domain and then send your request to it. Hope it will help.

**Comment by ArturSS7** [comment]
By the way, logs are being done on the registered domains and on all the domains 1 level higher then the registered. For example you have generated jdjfkdkk.e.example.com. Everything on  jdjfkdkk.e.example.com will be logged and also everything on *. jdjfkdkk.e.example.com will be logged.
